test/bats.bats

```diff
@@ -221,14 +221,41 @@ fixtures bats
   [ $status -eq 0 ]
 }
 
-@test "load supports scripts on the PATH" {
+@test "load supports scripts on the BATS_LIB_PATH" {
   path_dir="$BATS_TMPNAME/path"
```

Comment:
Add a couple spaces to `path_dir` to test the splitting code. Repeat with the tests following.

Reply:
Good idea, I've added a test specifically for whitespace splitting.